pkgs/by-name/go/gollama/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "gollama";
+  version = "1.27.17";
+
+  src = fetchFromGitHub {
+    owner = "sammcj";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-/KemOJwVHdb2BJnV53EVvbuE+0s3WOj4kzcox8hRZ6w=";
+  };
+
+  vendorHash = "sha256-MbadjPW9Oq3lRVa+Qcq4GXaZnBL0n6qLh5I2hJ0XhaY=";
+
+  doCheck = false;
+
+  ldFlags = [
+    "-s"
+    "-w"
+  ];
+
```

Comment:
Would be better to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook